webapp/components/experiments/Experiments.js

```diff
@@ -32,17 +52,89 @@ const Experiments = (props) => {
     props.viewExperiment(payload);
   };
 
+  const handleDelete = () => {
+    console.info('You clicked the delete icon.');
+  };
+
+  const handleClick = () => {
+    console.info('You clicked the Chip.');
+  };
+
+  const formatDate = (timestamp) => {
+    const date = new Date(timestamp)
+    return date?.toLocaleDateString().replaceAll('/', '.');
+  }
+
   return (
-    <div>
-      <TextField />
-      <Button onClick={editExperiment} />
-      <Button onClick={cleanExperiment} />
-      <Button onClick={deleteExperiment} />
-      <Button onClick={viewExperiment} />
-      {
-        experiments.map((experiment) => <div>{experiment.name}</div>)
-      }
-    </div>
+    <GridLayout>
+      <div>
+        <Box>
+          <TableContainer component={Paper} className="experimentTableContainer">
+            <Table aria-label="simple table" className="experimentTable">
+              <TableBody>
+                { experiments.map((experiment) => (
+                  <TableRow key={experiment?.name}>
+                    <TableCell component="th" scope="row">
+                      <Button>
+                        <Typography variant="h6" className="experimentHead">
+                          {experiment?.name}
+                        </Typography>
+                        <ChevronRightIcon className="experimentHeadIcon" />
+                      </Button>
+                    </TableCell>
+                    <TableCell align="left">
+                      <Typography variant="h6" className="experimentDate">
+                        { formatDate(experiment?.timestamp) }
+                      </Typography>
+                    </TableCell>
+                    <TableCell align="left" className="experimentTableCell">
+                      {experiment?.state !== 'SIMULATING' ? 
```

Comment:
Would suggest to extract the states to constants since they will be reused repeatedly (similar to constants.js::MODEL_STATE)